README.md

```diff
@@ -36,3 +36,16 @@ A link to the google calendar item can be found [here](https://calendar.google.c
 We use https://hackmd.io/ for taking meeting minutes and will upload the resultant markdown file after the meeting has concluded.
 
 There is a template provided in [`misc/DEV_MEETING_TEMPLATE.md`](https://github.com/conda-forge/conda-forge.github.io/tree/main/misc/DEV_MEETING_TEMPLATE.md) that you should use to create a new hackmd document.
+
+## Code Of Conduct
+All are welcome to participate in conda-forge community (including the dev meetings) as long as the tenets of this Code of Conduct are adhered to:
```

Comment:
```suggestion
Everyone is welcome to participate in conda-forge community (including the dev meetings) as long as the tenets of this Code of Conduct are adhered to:
```

What do you think is more inclusive: "all are" or "everyone is"? Just pointing it out for you to decide :)